test/match-test.js

```diff
@@ -440,6 +440,52 @@ describe("sinonMatch", function () {
         });
     });
 
+    describe(".in", function () {
+        it("returns matcher", function () {
+            var inMatcher = sinonMatch.in([]);
+            assert(sinonMatch.isMatcher(inMatcher));
+        });
+
+        context("when the test is called with non array argument", function () {
```

Comment:
2 things here:

* Use `describe` instead _when necessary_
* Only nest tests when there is additional/unique setup

In this instance, the nesting is not warranted. Just change the label/name on the test below.